decisions/log/0036-clarify-product-amount

```diff
@@ -0,0 +1,102 @@
+
+We exclude the mass of product packaging from the declaredUnitAmount  and productMassPerDeclaredUnit  attributes for the following reasons: 
+- Excluding mass of packaging is common practice when assessing the functional unit in LCA 
+- Data recipients who wish to conduct mass balance (validation of procured products by mass) will do so using the unpackaged product, hence the need for the mass of the unpackaged product 
+
+Excluding the mass of packaging does not however imply that the emissions due to packaging will be excluded from disclosure – packaging emissions transparency are being addressed in separate backlog item: https://flat-dollar-c04.notion.site/Provide-clarity-on-packaging-accounting-i-e-primary-secondary-and-tertiary-packaging-inclusion-ed6a3c17c99845c788e45b9d9fbfb5fa  
+
+## Examples 
+The below examples indicates how a PCF will be encoded in the data model in V2 vs. V3 (this proposal). 
+
+### 12l bottle of Bio-Ethanol
+
+| Current version 2                             | 12L Bottle of bio-Ethanol (with packaging) |
+|-----------------------------------------------|--------------------------------------------|
+| declaredUnit	                                | liter                                      |
+| unitaryProductAmount	                        | 12                                         |
+| pCfExcludingBiogenic (kgCO2e/declared unit)   | 1.5                                 |
+| productMassPerDeclaredUnit (kg/declared unit) | 0.789                               |
+
+
+| Proposed version 3                            | 12L Bottle of bio-Ethanol (with packaging) | 12L Bottle of bio-Ethanol (with packaging) |
+|-----------------------------------------------|---------|-------|
+| declaredUnitOfMeasurement                     | liter	  | piece |
+| declaredUnitAmount                            | 12      | 1     |
+| pCfExcludingBiogenic (kgCO2e/declared unit)	| 18      | 18    |  
+| productMassPerDeclaredUnit (kg/declared unit) | 9.468	  | 9.468 |
+
+
+### 2000kg car, 600,000 kgCO2e/car
+
+| Current version 2                             | PCF of a car |
+|-----------------------------------------------|--------------|
+| declaredUnit                                  | kilogram     |
+| unitaryProductAmount                          | 2000         |
+| pCfExcludingBiogenic (kgCO2e/declared unit)   | 300          |
+| productMassPerDeclaredUnit (kg/declared unit) | 1            |
+
+| Proposed version 3                            |              | Alternative |
+|-----------------------------------------------|--------------|-------------|
+| declaredUnitofMeasurement                     | kilogram     | piece       |
+| declaredUnitAmount                            | 2000         | 1           |
+| pCfExcludingBiogenic (kgCO2e/declared unit)   | 600000       | 600000      |
+| productMassPerDeclaredUnit (kg/declared unit) | 2000         | 2000        |
```

Comment:
Also great catch! I just reviewed the logic and fully agree. Should be 2000 kg once this proposal is adopted in v3, which will also require us to update the description of `productMassPerDeclaredUnit`. I fixed the excel and made a comment accordingly. The good news is that this fix will reduce the confusion also of `productMassPerDeclaredUnit`, because it admittedly felt "odd" to populate that field with a 1 if the declaredUnit is kg.